sandbox/host/src/sandbox.rs

```diff
@@ -479,10 +477,10 @@ enum BackendContext {
 impl BackendContext {
     pub fn new(backend: SandboxBackend) -> BackendContext {
         match backend {
-            SandboxBackend::Wasmi => BackendContext::Wasmi,
+            SandboxBackend::Wasmi => BackendContext::Wasmi(WasmiBackend::new()),
 
             #[cfg(not(feature = "host-sandbox"))]
-            SandboxBackend::TryWasmer => BackendContext::Wasmi,
+            SandboxBackend::TryWasmer => BackendContext::Wasmi(WasmiBackend::new()),
```

Comment:
not related to the PR itself, but why try warmer under the feature still cause wasmi?